spikes/bullets/Weapon.cpp

```diff
@@ -0,0 +1,35 @@
+#include "stdafx.h"
+#include "Weapon.h"
```

Comment:
Fire rate will be maintained by the individual weapons, at the moment we
only have test weapons so i did not implement a fire-rate, should be
implemented shortly after we finish the store/purchasing system.